configure.sh

```diff
@@ -1,37 +1,84 @@
 #!/bin/bash
 
+dark_mode=true
+dark_terminal=true
+change_wallpaper=true
+wallpaper_path=/home/amnesia/Persistent/wallpapers/wallpaper.jpg
+auto_sleep=true
+dim_screen=true
+disable_camera=true
+disable_microphone=true
+disable_location=true
+disable_usb_auto_mount=false
+disable_file_history=true
+set_mouse_speed=false
+mouse_speed=-0.6
+disable_wifi=false
+
 # dark mode
-gsettings set org.gnome.desktop.interface color-scheme 'prefer-dark'
+if [ "$dark_mode" = true ]; then
+  gsettings set org.gnome.desktop.interface color-scheme 'prefer-dark'
+fi
+
 # dark terminal
-gsettings set org.gnome.desktop.interface gtk-theme 'Adwaita-dark'
+if [ "$dark_terminal" = true ]; then
+  gsettings set org.gnome.desktop.interface gtk-theme 'Adwaita-dark'
+fi
+
 # set wallpaper
-gsettings set org.gnome.desktop.background picture-uri-dark /home/amnesia/Persistent/wallpapers/wallpaper.jpg
+if [ "$change_wallpaper" = true ]; then
+  gsettings set org.gnome.desktop.background picture-uri-dark "$wallpaper_path"
+fi
 
 # set auto sleep to never
-gsettings set org.gnome.desktop.session idle-delay 0
+if [ "$auto_sleep" = true ]; then
+  gsettings set org.gnome.desktop.session idle-delay 0
+fi
+
 # turn off automatic screen lock
-gsettings set org.gnome.desktop.screensaver lock-enabled false
+if [ "$dim_screen" = true ]; then
+  gsettings set org.gnome.desktop.screensaver lock-enabled false
```

Comment:
why using `dim_screen` variable for this? shouldn't you be using something like `auto_screen_lock`?